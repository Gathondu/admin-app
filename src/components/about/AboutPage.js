import React, {Component} from 'react';

class AboutPage extends Component {
    render() {
        return (
            <div>
                <h1>{this.props.title}</h1>
                <p>This application uses React, Redux, React Router and a variety of other helpful libraries.</p>
            </div>
        );
    }
}

export default AboutPage;