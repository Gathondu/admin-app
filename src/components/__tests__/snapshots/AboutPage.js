import renderer from 'react-test-renderer';
import AboutPage from "../../about/AboutPage";
import React from "react";

describe('The About Page Component', () => {
    it('It does not regress',() => {
        const title = 'About Test';
        const rendered = renderer.create(
            <AboutPage title={title} />
        );
        expect(rendered.toJSON()).toMatchSnapshot();
    });
});