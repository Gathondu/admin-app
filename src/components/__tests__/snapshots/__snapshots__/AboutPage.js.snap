// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The About Page Component It does not regress 1`] = `
<div>
  <h1>
    About
  </h1>
  <p>
    This application uses React, Redux, React Router and a variety of other helpful libraries.
  </p>
</div>
`;
